Include/internal/pycore_frame.h

```diff
@@ -26,6 +26,7 @@ struct _frame {
     char f_trace_lines;         /* Emit per-line trace events? */
     char f_trace_opcodes;       /* Emit per-opcode trace events? */
     char f_fast_as_locals;      /* Have the fast locals of this frame been converted to a dict? */
```

Comment:
Can `f_fast_as_locals` be dropped now? It was associated with the old dict conversion functions that are now no-ops.